Sources/OTel/OTelAPI/OTel+Bootstrap.swift

```diff
@@ -124,32 +124,57 @@ extension OTel {
         var services: [Service] = []
 
         if configuration.logs.enabled {
-            try services.append(bootstrapLogs(configuration: configuration))
+            try services.append(bootstrapLogs(configuration: configuration, logger: logger))
         }
         if configuration.metrics.enabled {
-            try services.append(bootstrapMetrics(configuration: configuration))
+            try services.append(bootstrapMetrics(configuration: configuration, logger: logger))
         }
         if configuration.traces.enabled {
-            try services.append(bootstrapTraces(configuration: configuration))
+            try services.append(bootstrapTraces(configuration: configuration, logger: logger))
         }
 
         return ServiceGroup(services: services, logger: logger)
     }
 
-    internal static func bootstrapTraces(configuration: OTel.Configuration) throws -> some Service {
+    internal static func bootstrapTraces(configuration: OTel.Configuration, logger: Logger) throws -> some Service {
         let backend = try makeTracingBackend(configuration: configuration)
         InstrumentationSystem.bootstrap(backend.factory)
         return backend.service
     }
 
-    internal static func bootstrapMetrics(configuration: OTel.Configuration) throws -> some Service {
+    internal static func bootstrapMetrics(configuration: OTel.Configuration, logger: Logger) throws -> some Service {
```

Comment:
Why did bootstrap{Traces/Metrics} also gain the logger parameter? Mainly for symmetry, or is there a future usecase?

Reply:
There's upcoming work to add a bunch of diagnostic logging. And so I added them here for now for symmetry, yes.